libs/shared/ui-components/src/SocialCard/types.ts

```diff
@@ -0,0 +1,82 @@
+import { DomainVariant } from '@quansight/shared/types';
+
+type TSocialCardPropsBase = {
+  /**
+   * A concise title for the related content.
+   * Platform specific behaviors for twitter:
+   *  - iOS, Android: Truncated to two lines in timeline and expanded Tweet
+   *  - Web: Truncated to one line in timeline and expanded Tweet
+   * More info: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary
+   */
+  title: string;
+
+  /**
+   * You should not re-use the title as the description or use this field to describe the general services provided by the website.
+   * Platform specific behaviors for twitter:
+   *  - iOS, Android: Not displayed
+   *  - Web: Truncated to three lines in timeline and expanded Tweet
+   * More info: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image
+   */
+  description: string;
+
+  /**
+   * Uses DomainVariant enum values if the optional properties are not added
+   * SocialCard component uses it to set default images and alt based on the variant value.
+   */
+  variant: DomainVariant;
+
+  /**
+   * The `twitterLargeImage` prop controls whether the large or small Twitter summary card will be used.
+   * The `SocialCard` component *should* be configured with a `true` default value for this prop.
+   * Remember to use the twitter:image that fits aspect ratio of 2:1 when twitterLargeImage is true.
+   * with minimum dimensions of 300x157 or maximum of 4096x4096 pixels (see twitterImage type above for more details).
+   * full-width prominent image alongside a tweet. It is designed to give the reader a rich photo experience,
+   * and clicking on the image brings the user to your website.
+   * More info: `https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image`
+   * Our default Twitter cards for Quansight and Quansight Labs are large
+   */
+  twitterLargeImage?: true;
```

Comment:
@Surafel-Getachew -- With this moved to the base class, we'll need those 1:1 logo images for the small Twitter images, and to add selection code for them into `SocialCard.tsx:defaultTwitterImage`, right?

Reply:
Or, we could move `twitterLargeImage` back into `TSocialCardPropsCustomizedImage` -- I think this might actually have been what @gabalafou meant to do.

Reply:
Ah, @bskinn you are right WDYT should we revert back?

Reply:
Yeah, I would revert that back. Then, I think we're ready to merge.